pylint_django/compat.py

```diff
@@ -28,16 +29,3 @@
     # if not available, will be handled by the django_installed checker
     django_version = (1, 5)
```

Comment:
FIY we're Python 3 only ATM so no need to worry about older Django versions, right ?

Reply:
The hardcoded `1.5` was really just to try to avoid complete failures due to not being able to import django at all. I will look through this but I think this method can go completely too.